README.md

```diff
@@ -1,56 +1,36 @@
 Collection of tools to deploy IOHK infrastructure.
 
-### Structure
+### File structure
 
 - `deployments` - includes all NixOps deployments controlled via `.hs` scripts
 - `modules` - NixOS modules
 - `lib.nix` - wraps upstream `<nixpkgs/lib.nix>` with our common functions
 - `scripts` - has bash scripts not converted to Haskell/Turtle into Cardano.hs yet
 - `default.nix` - is a collection of Haskell packages
 - `static` includes files using in deployments
-- `jobsets` is used by Hydra CI
+- `jobsets` jobsets declaratively provisioned by Hydra CI
+- `tests` - NixOS tests for the `modules`
+- `iohk` - Haskell automation of opsy stuff
 
 
-### Compiling localy
+### Usage
 
-     $ ./CardanoCSL.hs build
+    $(nix-build -A iohk-ops)/bin/iohk-ops --help
 
-### Deploying changes
 
-    $ ./CardanoCSL.hs deploy
+### Deployments file structure
 
-**Note this will not remove machines if they no longer exist.**
+There is one global resource file `deployments/keypairs.nix` for all deployments. That's where
```

Comment:
That file doesn't exist on the target branch, but it does exist in `master`.

Reply:
Indeed, it has to be moved back.

Reply:
That should be done before this is merged.